tests/scalability/medic-os.sh

```diff
@@ -8,4 +8,5 @@ cd cht-infrastructure/self-hosting/prepare-system/ubuntu/;
 mkdir -p /home/medic/self-hosting/main/
 echo DOCKER_COUCHDB_ADMIN_PASSWORD=medicScalability >> /home/medic/self-hosting/.env
 echo HA_PASSWORD=medicScalability >> /home/medic/self-hosting/.env
-./prepare.sh
\ No newline at end of file
+./prepare.sh
+sudo shutdown -P +60
```

Comment:
Move the shutdown to the beginning of the script. That way if something fails before hand the shutdown is triggered and the ec2 instance won't just hang around. I think 60 minutes should be fine for now and if we run into issues we can bump it up.

Reply:
Done